tests/check_ncb64.c

```diff
@@ -106,6 +106,36 @@ START_TEST(check_nc_b64_enc_n_file_foobar)
 }
 END_TEST
 
+START_TEST(check_nc_b64_enc_n_file_long_text)
+{
+        char out[8000];
+        char result[8000];
+        unsigned long len = 0;
+        FILE *fh = NULL;
+        size_t n = 8000 - 1;
+        const char *filename = TOPSRCDIR "/tests/nc_b64enc_test_files/long_text";
+        const char *resultfile = TOPSRCDIR "/tests/nc_b64enc_test_files/long_text.b64";
+
+        fh = fopen(resultfile, "r");
+
+        fseek(fh, 0, SEEK_END);
+        len = (unsigned long) ftell(fh);
+        fseek(fh, 0, SEEK_SET);
+
+        if (fread(result, len, 1, fh) == 0) {
+            ck_abort_msg("Result file [long_text.b64] should not be 0 length");
```

Comment:
A zero return here would happen if the file length was shortened in the race between getting the length and reading the data, but the file could still have non zero length.